script.js

```diff
@@ -9,3 +10,211 @@ document.querySelectorAll('.head-link').forEach((n) => n.addEventListener('click
   hamburger.classList.remove('active');
   menuBar.classList.remove('show-navbar');
 }));
+
+const projectsCards = [
+  {
+    id: '0',
+    name: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry standards',
+    cardSkills: ['html', 'bootstrap', 'Ruby'],
+  },
+  {
+    id: '1',
+    name: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    cardSkills: ['html', 'bootstrap', 'Ruby'],
+  },
+  {
+    id: '2',
+    name: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    cardSkills: ['html', 'bootstrap', 'Ruby'],
+  },
+  {
+    id: '3',
+    name: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    cardSkills: ['html', 'bootstrap', 'Ruby'],
+  },
+  {
+    id: '4',
+    name: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    cardSkills: ['html', 'bootstrap', 'Ruby'],
+  },
+  {
+    id: '5',
+    name: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    cardSkills: ['html', 'bootstrap', 'Ruby'],
+  },
+];
+
+const projectsContainer = document.getElementById('frame-sub');
+
+projectsCards.forEach((projects, index) => {
```

Comment:
Please align the project name and close the button horizontally on the mobile view. The text of the technologies list is overflowing out of the box. Pop-up cards should be responsive and should be displayed according to the Figma design.
![align](https://user-images.githubusercontent.com/35504149/197001450-2fd4937f-ea2a-41f6-8db4-0c21a0ed7700.png)